src/components/map/CuriosityMap.tsx

```diff
@@ -1,9 +1,26 @@
-import React from "react";
+import React, {useState} from "react";
+import './Map.scss'
```

Comment:
Also, this file should be named the same as the component it refers to:
```suggestion
import './CuriosityMap.scss'
```

Reply:
Unless it's going to be shared by multiple components, but in which case you should have a single component that imports all of them anyway.

In fact, why do you have separate components for all three maps? We should talk about this